db/db.ts

```diff
@@ -204,3 +208,63 @@ export const getPublishedExplorersBySlug = async (
         return keyBy(processed, "slug")
     })
 }
+
+export const getLatestDataInsights = (
+    limit = 5
+): Promise<MinimalDataInsightInterface[]> => {
+    return knexRaw(
+        `
+        SELECT
+            content->>'$.title' AS title,
+            publishedAt,
+            ROW_NUMBER() OVER (ORDER BY publishedAt DESC) - 1 AS \`index\`
```

Comment:
seems a bit heavy handed, given that we can get the index in the `map()` below (EDIT: I'm seeing now this has been merely moved so not really part of this PR)